frontend/src/App.css

```diff
@@ -50,6 +50,7 @@ html, body {
   display: flex;
   flex-direction: column;
   min-height: 100vh;
+  background-color: #FFFFFF;
```

Comment:
Adding background-color: #FFFFFF; directly to html, body enforces a white background app-wide and can break dark-mode or theming expectations and cause visual regression across many pages.

Use a CSS variable or root class for theming and avoid forcing white globally. Example: 
```
:root { 
--bg-color: #FFFFFF; 
} 
html, body { 
background-color: var(--bg-color); 
} 
```
Or restrict to .app-container instead of html/body if intended locally.